### PR TITLE
Add crate metadata

### PR DESCRIPTION
**Pull request by rnestler**
This is needed to publish to crates.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,21 @@ Start the service:
...
+ * MIT license ([LICENSE-MIT](LICENSE-MIT) or
+   http://opensource.org/licenses/MIT)
+
+at your option.
+
+### Contribution
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the work by you, as defined in the Apache-2.0 license, shall
+be dual licensed as above, without any additional terms or conditions.
```
Comment (dbrgn) [inline README.md:46]: Ah, cool. A mini-CLA :) I doubt it's legally binding, but still.

**Review by dbrgn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rnestler) [reply README.md:46]: I copied it from [spaceapi-rs](https://github.com/coredump-ch/spaceapi-rs/blob/master/README.md#contribution)...

File: validator/Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [package]
 name = "spaceapi-validator"
 version = "0.1.0"
+repository = "https://github.com/spacedirectory/validator"
 authors = ["Raphael Nestler <raphael.nestler@gmail.com>"]
+description = "Validator for Space API JSON data."
+keywords = ["spaceapi", "validator", "hackerspaces", "status", "api"]
+license = "MIT OR Apache-2.0"
```
Comment (rnestler) [inline validator/Cargo.toml]: Hmm according to http://doc.crates.io/manifest.html#package-metadata it should be MIT/Apache-2.0

Comment (dbrgn) [reply validator/Cargo.toml]: You're right. Change it! :)

Comment (rnestler) [reply validator/Cargo.toml]: I need to check, but we may need to change the spaceapi-* crates as well.

**Comment by dbrgn** [comment]
You should either fix your last commit msg or squash the commits :)

**Comment by rnestler** [comment]
> You should either fix your last commit msg or squash the commits :)

Did you mean the small typo? 's' and 't' are very close in the alphabet 😉

**Pull request merged**